### DISCUSSION TITLE
Monogame 3.8.2 iOS does not build

### DISCUSSION THREAD

**Discussion by kurtcz** [MonoGame/MonoGame#8509] [iOS / iPadOS]
I tried to update my Monogame 3.8.1 iOS and Android apps to 3.8.2 but unfortunately the iOS app does not build. Android version builds fine. I am using Monogame.Framework.IOS 3.8.2.1105 and Monogame.Content.Builder.Task 3.8.2.1105 So whilst investigating I tried to build an empty Monogame.iOS project from template.

```
dotnet new mgios -o appname
cd appname
dotnet build
```
but I get the same error:
```
error CS0234: The type or namespace name 'Xna' does not exist in the namespace 'Microsoft'
```

**Comment by Asthegor** [comment]
I had the same issue on Windows after updating MonoGame.Framework.DesktopGL and MonoGame.Content.Builder.Task from 3.8.1.303 to 3.8.2.1105.

I had my dotnet tools installed as global.
After I follow this tutorial : https://community.monogame.net/t/resolving-mgcb-editor-not-found-error/19419
1. Uninstall all MonoGame dotnet tools (global)
2. Changing the version in the .config/dotnet-tools.json in the project folder
3. Removing the folder %USERPROFILE%\.nuget\packages
4. Closing VS2022 (latest version)
5. Installing packages from NuGet via CLI
=> download the packages monogame.framework.desktopgl and monogame.content.builder.task 3.8.2.1105
6. Open the solution in VS2022
7. Compile the project
=> the project download and install the dotnet tools (dotnet-mgcb, dotnet-mgcb-editor, dotnet-mgcb-editor-linux, dotnet-mgcb-editor-windows, dotnet-mgcb-editor-mac)

The error CS0234 remains.

Workaround : downgrade to the previous stable version (aka 3.8.1.303)
